### ISSUE TITLE
Having multiple SignalR connection using abp.signalr

### ISSUE THREAD

**Issue by mohammad-shadmehr** [aspnetboilerplate/module-zero-core-template#190]
I have got a project which is using ASPZero template. I have successfully integrated SignalR into my solution and the real time notification works fine. What I want is adding another hub into my solution or extending the existing signalr hub ad a more to it. In the Signalr document it says to add the following to the startup file:

```
app.UseSignalR(routes =>
{
     routes.MapHub<AbpCommonHub>("/signalr"); //default hub
     routes.MapHub<HitchNotification.HitchHub>("/hitchHub"); //my hub
});
```
I added mine next to it. The issue however is in the client where I need to setup the connection through the abp.signalr! In the SignalRAspNetCoreHelper it sets the url to use the "signalr" hub (default one).
```

export class SignalRAspNetCoreHelper {
static initSignalR(): void {

    var encryptedAuthToken = new UtilsService().getCookieValue(AppConsts.authorization.encrptedAuthTokenName);

    abp.signalr = {
        autoConnect: true,
        connect: undefined,
        hubs: undefined,
        qs: AppConsts.authorization.encrptedAuthTokenName + "=" + encodeURIComponent(encryptedAuthToken),
        url: AppConsts.remoteServiceBaseUrl + '/signalr'
    };

    jQuery.getScript(AppConsts.appBaseUrl + '/assets/abp/abp.signalr-client.js');
}
```

If I change the "signalr" to "hitchHub" it works fine but i actually want both in my application! I also tried to create a Helper for my own hub similar to the SignalRAspNetCoreHelper and initialize it in the app.component:

```
ngOnInit(): void {
    if (this.appSession.application && this.appSession.application.features['SignalR']) {
        if (this.appSession.application.features['SignalR.AspNetCore']) {
            SignalRAspNetCoreHelper.initSignalR();
            HitchHubHelper.initHitchHub();  
        } 
    }
}
```
but seems that the abp.signalr can't have multiple connection to different hubs.

So, basically I have got 2 questions:

1. Is there anyway that I could add my own hub's functions to the existing default AbpCommonHub? that way I could simply modify the abp.signalr-client file
2. If the above is impossible, how can I have multiple hubs on the abp.signalr to be accessible anywhere within my application?
